delfin/drivers/fujitsu/eternus/cli_handler.py

```diff
@@ -0,0 +1,412 @@
+import hashlib
+import re
+import threading
+
+import six
+from oslo_log import log
+from oslo_utils import units
+
+from delfin import exception
+from delfin.common import constants
+from delfin.drivers.fujitsu.eternus import consts
+from delfin.drivers.fujitsu.eternus.consts import DIGITAL_CONSTANT
+from delfin.drivers.fujitsu.eternus.eternus_ssh_client import \
+    EternusSSHPool
+from delfin.drivers.utils.tools import Tools
+
+LOG = log.getLogger(__name__)
+
+
+class CliHandler(object):
+    lock = None
+
+    def __init__(self, **kwargs):
+        self.lock = threading.RLock()
+        self.kwargs = kwargs
+        self.ssh_pool = EternusSSHPool(**kwargs)
+
+    def login(self):
+        """Test SSH connection """
+        try:
+            self.exec_command(consts.GET_STORAGE_STATUS)
+        except Exception as e:
+            error = six.text_type(e)
+            LOG.error("Login error: %s", error)
+            raise exception.SSHException(error)
```

Comment:
Why we raise `SSHException`, it is suggested to use `raise e` .

Reply:
Has been modified